ansible/roles/debops.postfix/defaults/main.yml

```diff
@@ -1357,5 +1792,71 @@ postfix__ferm__dependent_rules:
                     if ("submission" in postfix__env_active_services|d([]))
                     else "absent" }}'
                                                                    # ]]]
+# .. envvar:: postfix__ldap__dependent_tasks [[[
+#
+# Configuration for the :ref:`debops.ldap` Ansible role.
+postfix__ldap__dependent_tasks:
+
+  - name: 'Create Postfix account for {{ postfix__ldap_device_dn | join(",") }}'
+    dn: '{{ postfix__ldap_binddn }}'
+    objectClass: '{{ postfix__ldap_self_object_classes }}'
+    attributes: '{{ postfix__ldap_self_attributes }}'
+    no_log: '{{ postfix__no_log|d(True) }}'
+    state: '{{ "present"
+                if (postfix__ldap_enabled|bool and
+                    postfix__ldap_device_dn|d())
+                else "ignore" }}'
+
+  - name: 'Create Postfix group container for {{ postfix__ldap_device_dn | join(",") }}'
+    dn: '{{ postfix__ldap_groups_dn }}'
+    objectClass: 'organizationalUnit'
+    attributes:
+      ou: '{{ postfix__ldap_groups_rdn.split("=")[1] }}'
+      description: 'User groups used in Postfix'
+    state: '{{ "present"
+               if (postfix__ldap_enabled|bool and
+                   postfix__ldap_device_dn|d() and
+                   postfix__ldap_private_groups|bool)
+               else "ignore" }}'
+
+  - name: 'Create Postfix domains container for {{ postfix__ldap_device_dn | join(",") }}'
+    dn: '{{ postfix__ldap_domains_dn }}'
+    objectClass: 'organizationalUnit'
+    attributes:
+      ou: '{{ postfix__ldap_domains_rdn.split("=")[1] }}'
+      description: 'Virtual Mail Domains used in Postfix'
+    state: '{{ "present"
+               if (postfix__ldap_enabled|bool and
+                   postfix__ldap_device_dn|d() and
+                   postfix__ldap_private_groups|bool)
+               else "ignore" }}'
+
+  - name: 'Create Postfix Default Virtual Mail Domain for {{ postfix__ldap_default_virtual_domain_dn }}'
+    dn: '{{ postfix__ldap_default_virtual_domain_dn }}'
+    objectClass: 'dNSDomain'
+    state: '{{ "present"
+               if (postfix__ldap_enabled|bool and
+                   postfix__ldap_device_dn|d() and
+                   postfix__ldap_private_groups|bool)
+               else "ignore" }}'
+
+
+                                                                   # ]]]
+# .. envvar:: postfix__ldap_dependent_system_users [[[
+#
+# Configuration for the :ref:`debops.system_users` Ansible role.
+postfix__ldap_dependent_system_users:
+
+  - name: '{{ postfix__virtual_mail_posix_user }}'
+    group: '{{ postfix__virtual_mail_posix_group }}'
+    uid: '{{ postfix__virtual_mail_posix_uidnumber }}'
+    gid: '{{ postfix__virtual_mail_posix_gidnumber }}'
+    system: True
+    comment: 'Postfix Virtual Mail user'
+    shell: '/usr/sbin/nologin'
+    home: '{{ postfix__virtual_mailbox_base }}'
+    skeleton: null
```

Comment:
This is an interesting trick! Perhaps with this various UNIX account home directories created for system services by DebOps would not be littered with stuff from `/etc/skel/`. Thanks, I'll look into using it more.